src/components/navigation/header/drawer/userInfo/index.tsx

```diff
@@ -48,29 +53,26 @@ const DrawerUserInfo: FC<Props> = ({ user }) => {
           </Stack>
         </Stack>
         {selectedTenant ? (
-          <Stack direction="row">
-            <GarageIcon />
-            <Stack
-              direction={{
-                base: 'column',
-                md: 'row',
-              }}
-              spacing={{
-                base: 0,
-                md: 'sm',
-              }}
-            >
-              <Box as="span" fontWeight="bold">
-                {selectedTenant.billingName || selectedTenant.id}
-              </Box>
-              <Box as="span">
-                {selectedTenant.billingZipCode} {selectedTenant.billingCity}
-              </Box>
-            </Stack>
-          </Stack>
+          <>
+            <Show above="sm">
+              <SelectedTenantInfo selectedTenant={selectedTenant} />
+            </Show>
+            <Show below="xs" marginX="-lg" marginBottom="-lg">
+              <TenantSelectionMenu
+                user={user}
+                selectedTenant={selectedTenant}
+                selectTenant={selectTenant}
+              />
+            </Show>
+          </>
         ) : null}
       </Stack>
-      <Divider />
+      <Divider
+        borderBottomWidth={{
+          base: selectedTenant ? 0 : '1px',
```

Comment:
The accordion bottom border always sticks due to the last of type pseudo selector, so let's hide the divider